crowbar_framework/app/models/crowbar_service.rb

```diff
@@ -309,7 +322,10 @@ def apply_role_pre_chef_call(old_role, role, all_nodes)
   def apply_role (role, inst, in_queue)
     @logger.debug("Crowbar apply_role: enter")
     answer = super
-    @logger.debug("Crowbar apply_role: super apply_role finished")
+    if answer.first != 200
+      @logger.error("Crowbar apply_role: super apply_role finished with error")
+      return answer
+    end
```

Comment:
Just wondering (and you probably know better since you looked at such logs recently): do we need to keep the debug log if we got a `200`?